include/crpropa/Module.h

```diff
@@ -59,7 +59,17 @@ class AbstractCondition: public Module {
 	void setMakeAcceptedInactive(bool makeInactive);
 	void setRejectFlag(std::string key, std::string value);
 	void setAcceptFlag(std::string key, std::string value);
+
+	void printRejectFlag();
+	void printAcceptFlag();
+};
+
+class DeactivateModule: public AbstractCondition {
```

Comment:
Is it used somewhere or just a general module to deactivate the candidate? 

In this case I would simply call it `Deactivation` as we use nouns for other modules, too, e.g. `MaximumTrajectoryLength`, etc.

Please also add short comment for documentation.

Reply:
This is a general module, also intended for debugging. This makes it easier to reproduce the original error as they are faster in deactivating the candidate. 

I will rename it.